contributors.txt

```diff
@@ -64,3 +64,4 @@ YYYY/MM/DD, github id, Full name, email
 2019/01/24, IEVin, Ivan Vinogradov, i.e.vin@ya.ru
 2019/01/29, maraf, Marek Fišera, mara@neptuo.com
 2019/01/30, stivx0, Ivan Sterkhov, stivx0@gmail.com
+2019/01/30, markonius, Marko Bakšić, markonius@protonmail.ch
```

Comment:
To prevent contention between other pull requests, create a separate pull request that does just this..  Once they have the cla bot in place, we won't have to worry about this.

Reply:
No.  Base the pr off of master.  Look at the other pull requests like it.

Reply:
Done